align/pnr/placer.py

```diff
@@ -39,6 +39,7 @@ def place( *, DB, opath, fpath, numLayout, effort, idx, lambda_coeff, select_in_
     hyper.LAMBDA = lambda_coeff
     hyper.use_analytical_placer = use_analytical_placer
     hyper.use_ILP_placer = place_using_ILP
+    hyper.ILP_runtime_limit = placer_ilp_runtime
```

Comment:
Could you please add comment to explain what this parameter does, what the unit is, etc.?